pkg/cmd/cli/cmd/create/clusterquota.go

```diff
@@ -70,15 +70,31 @@ func (o *CreateClusterQuotaOptions) Complete(cmd *cobra.Command, f *clientcmd.Fa
 		return fmt.Errorf("NAME is required: %v", args)
 	}
 
-	selector, err := unversioned.ParseToLabelSelector(cmdutil.GetFlagString(cmd, "project-selector"))
+	var labelSelector *unversioned.LabelSelector
+	labelSelectorString := cmdutil.GetFlagString(cmd, "project-label-selector")
+	if len(labelSelectorString) > 0 {
+		var err error
+		labelSelector, err = unversioned.ParseToLabelSelector(labelSelectorString)
+		if err != nil {
+			return err
+		}
+	}
+
+	annotationSelector, err := unversioned.ParseToLabelSelector(cmdutil.GetFlagString(cmd, "project-annotation-selector"))
```

Comment:
> Will an empty string create an empty selector?

It will create one with zero `MatchLabels` in it, which is treated as empty.  Empty is allowed as long as the label selector is provided.